brats/transformations.py

```diff
@@ -73,3 +78,77 @@ def hist_match(source: np.ndarray, template: np.ndarray) -> np.ndarray:
                                     template_values)
         result[positive] = interp_t_values[bin_idx]
         return result
+
+
+class NiftiOrderTransformation:
+    """
+    Changes dimensions order from nifti (H,W,D,C) to torch convention (C,D,W,H).
+    It can be applied to both ``torch.Tensor`` or ``numpy.ndarray``.
+    """
+
+    def __call__(self, img: typing.Union[np.ndarray, torch.Tensor]) -> typing.Union[np.ndarray, torch.Tensor]:
+        assert len(img.shape) == 4
+
+        if isinstance(img, torch.Tensor):
```

Comment:
Python provides its own tools for dispatching on single type. Maybe consider using them.
https://docs.python.org/3.8/library/functools.html?highlight=dispatch#functools.singledispatch

Reply:
I changed it to single dispatch approach. 
NOTE: In Python 3.8 You can use `singledispatchmethod` on methods: 
https://docs.python.org/3.8/library/functools.html?highlight=dispatch#functools.singledispatchmethod
We will not use python 3.8 in this project, but I wanted to let you know.

Reply:
Thanks for the info! :D